### ISSUE TITLE
[MU4 Issue] Underscores that appear when you press ALT are too close to the text (Windows and Linux)

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#12467]
**To Reproduce**
1. Open MU
2. Press Alt 
3. Look at the File menu - **underscores are too close to the text** (comparing with MU3 and other apps (Firefox))

![Capture](https://user-images.githubusercontent.com/90187801/179732771-0bb9e8c4-8a37-4e55-b3da-ee3a8b3961c7.PNG)

**Screenshots**

**Platform information**
Windows10/Linux Ubuntu

**Additional context**
Doesn't occur in **MU3**

**Comment by krasko78** [comment]
This looks much better now (Qt 6.3.0 and probably 6.8.0 too).

**Comment by DmitryArefiev** [comment]
Yeah, it looks better now! It could be more gap between letter and underscore, but I think it's very minor. 

![1](https://github.com/user-attachments/assets/c64b56b1-7bc6-4db4-b276-50c9eac2e24e)


The issue became better in **4.4.0** when we switched to **Qt6 (6.2.4)**

In **Qt6.8** builds (#25016) it also looks fine. 
But I've checked only on Linux. There are **problems with running windows build** from  #25016 (FYI @cbjeukendrup )
